doc/ca65.sgml

```diff
@@ -1278,6 +1278,35 @@ writable.
 Pseudo functions expect their arguments in parenthesis, and they have a result,
 either a string or an expression.
 
+<sect1><tt>.ADDRSIZE</tt><label id=".ADDRSIZE"><p>
+
+  The <tt/.ADDRSIZE/ function is used to return the interal address size 
+  associated with a symbol. This can be helpful in macros when knowing the address 
+  size of symbol can allow for custom instructions.
+
+  Example:
+
+  <tscreen><verb>
+        .macro myLDA foo
+
+  	        .if .ADDRSIZE(foo) = 1
+  	          	        ;do custom command based on zeropage addressing:
+  	          	        .byte .A5h, foo
```

Comment:
Now, it's my turn "to pick a nit". ;-)

That looks, to me, like too much indentation.